agent/acs/model/api/api-2.json

```diff
@@ -669,7 +676,8 @@
         "name":{"shape":"String"},
         "type":{"shape":"VolumeType"},
```

Comment:
`VolumeType` should change to add 'efs' option.

Reply:
good catch!!